views/admin/applicationview.ejs

```diff
@@ -0,0 +1,23 @@
+<%- include('../partials/head.ejs') %>
+
+<div>
+    <h2 class="text">Aplication from:</h2>
```

Comment:
dlaczego wszystko ma klasę text??

Reply:
jkc to w tych komentarzach też można wyjaśniać pierdoły, to, że coś napisałem, nie znaczy, że coś jest źle

Reply:
Klasa Text była użyta przy wyświetlaniu textu w cssie post editora a tu jest praktycznie to samo

Reply:
ta klasa technicznie nie jest już używana ale jakby trzeba było zmienić czcionke w całym projekcie to imo dobrze żeby była